client/src/test/java/com/paypal/selion/mobile/sample/TestTapPage.java

```diff
@@ -0,0 +1,61 @@
+/*-------------------------------------------------------------------------------------------------------------------*\
+|  Copyright (C) 2016 PayPal                                                                                          |
+|                                                                                                                     |
+|  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance     |
+|  with the License.                                                                                                  |
+|                                                                                                                     |
+|  You may obtain a copy of the License at                                                                            |
+|                                                                                                                     |
+|       http://www.apache.org/licenses/LICENSE-2.0                                                                    |
+|                                                                                                                     |
+|  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed   |
+|  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for  |
+|  the specific language governing permissions and limitations under the License.                                     |
+\*-------------------------------------------------------------------------------------------------------------------*/
+
+package com.paypal.selion.mobile.sample;
+
+import com.paypal.selion.annotations.MobileTest;
+import com.paypal.selion.internal.platform.grid.WebDriverPlatform;
+import com.paypal.selion.logger.SeLionLogger;
+import com.paypal.selion.mobile.sample.mobile.SamplePage;
+import com.paypal.selion.mobile.sample.mobile.TapPage;
+import com.paypal.selion.platform.mobile.ios.UIAButton;
+import com.paypal.test.utilities.logging.SimpleLogger;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TestTapPage {
```

Comment:
@alizelzele how do I run this test class? I tried with `-DSELION_MOBILE_PLATFORM=IOS -DSELION_MOBILE_NODE_TYPE=appium`